src/hazmat/float.rs

```diff
@@ -1,5 +1,10 @@
 //! Const-context floating point functions that are currently not present in `core`.
 
+use core::f64;
+use crypto_bigint::Uint;
+#[allow(unused_imports)]
```

Comment:
Why is this needed? Clippy can't tell the trait is being used?

Reply:
Tests and clippy do not need it, but actually building the library does, so this is here to shut up the warning.